cdc-accelerator/feature/partner/permissionGroups.js

```diff
@@ -33,10 +33,13 @@ export default class PermissionGroups extends PartnerFeature {
         if (permissionGroupsRes.errorCode) {
             throw new Error(JSON.stringify(permissionGroupsRes))
         }
-        fs.writeFileSync(
-            path.join(featureDirectory, PermissionGroups.PERMISSIONGROUP_FILE_NAME),
-            JSON.stringify(this.remove_built_in_permission_groups(permissionGroupsRes.groups), null, 4),
-        )
+        if (Object.keys(permissionGroupsRes.groups) != '') {
```

Comment:
Please add a unit test to verify if this if works as expected.
Makes sense to call acl.init if the if is false?